app/javascript/components/ImageUploader.tsx

```diff
@@ -42,7 +43,10 @@ export const ImageUploader = ({
           <LoadingSpinner className="size-8" />
         </Placeholder>
       ) : imageUrl == null ? (
-        <Placeholder className="aspect-square items-center" style={{ background }}>
+        <Placeholder
+          className={classNames("aspect-square items-center", background && "bg-transparent")}
```

Comment:
Placeholder has a default background, and Tailwind background colours also set `background-blend-mode` which messes with the opacity we set in the `background` style